xbmc/guilib/GUIIncludes.h

```diff
@@ -47,13 +47,13 @@ class CGUIIncludes
   void Clear();
 
   /*!
-   \brief Load all include components (defaults, constants, variables, expressions and includes)
-    from the given \code{file}.
-   
-   \param file the file to load
-   \return true if the file was loaded otherwise false
-   */
-  bool Load(const std::string &file);
+  \brief Load all include components (defaults, constants, variables, expressions and includes)
+  from the main entrypoint \code{file}. Flattens nested expressions and expressions in variable
+  conditions after loading all other included files.
+
+  \param file the file to load
+  */
+  void LoadEntrypoint(const std::string &file);
```

Comment:
IMO no need to introduce a new method, just put the flatten method calls at the bottom of the `Load` method.